tests/components/counter/test_init.py

```diff
@@ -224,7 +243,10 @@ def test_no_initial_state_and_no_restore_state(hass):
 
 async def test_counter_context(hass, hass_admin_user):
     """Test that counter context works."""
-    assert await async_setup_component(hass, "counter", {"counter": {"test": {}}})
+    assert await async_setup_component(hass, "counter",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)